lib/src/component/abstract_transition_props.dart

```diff
@@ -25,6 +25,13 @@ mixin TransitionPropsMixin on UiProps {
   static final TransitionPropsMixin defaultProps = TransitionPropsMapView({})
     ..transitionCount = 1;
 
+  /// Prop that allows you to control the visibility of the component with props.
+  ///
+  /// __NOTE:__ In order for this work properly, if you have an implementation
+  /// of [AbstractTransitionComponent.initiallyShown] in your subclass - it must
+  /// use/reference this value.
+  bool shown;
```

Comment:
Named this way instead of `isShown` to avoid gross prop namespacing / forwarding issues in consumer libs that have an `isShown` prop in a mixin that is used in props classes that also mix in `TransitionPropsMixin`.